keps/sig-windows/1981-windows-privileged-container-support/README.md

```diff
@@ -809,7 +839,6 @@ _This section must be completed when targeting beta graduation to a release._
       - Impact of its outage on the feature:
       - Impact of its degraded performance or high-error rates on the feature:
 
-
 ### Scalability
```

Comment:
should consider constraints on networking for pods that use host network?

Reply:
I don't follow.

Reply:
there was some conversation when we went to sig-networking around scalability issues if the containers get attached to the host network.  Will this cause networking limitations if too many pods are scheduled with host networking enabled?

Reply:
Ah - I have absolutely no idea.
@dcantah, @JocelynBerrendonner any ideas?
Scalability requirements / concerns can be part of a push to beta

Reply:
There's essentially *no* network setup here, the question could be framed as "Is there a networking scalability concern with too many processes on Windows?" Jocelyn can chime in here though also